service/history/task/transfer_active_task_executor.go

```diff
@@ -291,6 +299,23 @@ func (t *transferActiveTaskExecutor) processDecisionTask(
 	return err
 }
 
+func (t *transferActiveTaskExecutor) allowTask(task *persistence.TransferTaskInfo) bool {
+	domainName, err := t.shard.GetDomainCache().GetDomainName(task.DomainID)
+	if err != nil {
+		t.logger.Error("Error when getting domain name",
```

Comment:
please cover this case in tests. new line coverage should be > 85% :) 
https://app.codecov.io/gh/uber/cadence/pull/5804?dropdown=coverage&src=pr&el=continue&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=uber#6777bf9ed3954eec1e8ba9682a9abe13-R305

Reply:
added more tests